catalog/ACA/catalog.jsonnet

```diff
@@ -19,5 +19,6 @@ local self_url = base_url + base_filename;
     ee.link.parent(parent_url),
     ee.link.self_link(self_url),
     ee.link.child_collection('ACA_reef_habitat_v1_0', base_url),
+    ee.link.child_collection('ACA_reef_habitat_v2_0', base_url),
```

Comment:
Please comment out this line and add a TODO as follows:

```javascript
// TODO(schwehr): Enable when the dataset is ready to be live.
// ee.link.child_collection('ACA_reef_habitat_v2_0', base_url),
```

Issue('ACA/reef_habitat/v2_0', 'ACA/ACA_reef_habitat_v2_0.json', 'parent_child', 'Child link when gee:skip_indexing is true: ACA/catalog